performance-tools/performance-cluster/scripts/library.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+
+RED='\033[00;31m'
```

Comment:
Some new functions but duplicating a lot what's already in `prow/scripts/library.sh`.